boto/compat.py

```diff
@@ -29,6 +29,13 @@
     import json
 
 
+# Use third party ordereddict for older versions of Python
+try:
+    from collections import OrderedDict
+except ImportError:
+    from ordereddict import OrderedDict
```

Comment:
Not sure if we want to have this logic here. From the looks of it ``boto ``does not have a dependency in ``setup.py`` for ``ordereddict`` so it may cause import errors... I would recommend just moving this to the script.